src/Relations/EmbedsMany.php

```diff
@@ -195,9 +196,11 @@ public function destroy($ids = [])
     /**
      * Delete all embedded models.
      *
+     * @param null $id
```

Comment:
Can we remove this argument, since it's not used.

Reply:
I know that it's not used, but it is part of the methods heading (name, arguments, etc.) as defined in the [parent method](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Database/Query/Builder.php#L3147).

Should we not keep the inherited methods heading the same, even though we might not use some arguments?

Reply:
Ok, then we need to add a comment to mention that the id is not used. And throw a LogicException if it is not null. We don't want to give the illusion that embeds can be deleted by Id. Even though it's an interesting feature to implement.

Reply:
The note must be on the argument description.
```suggestion
     * @param null $id Not used. Inherited from parent method.
```